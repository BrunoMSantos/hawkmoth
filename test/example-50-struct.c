/**
 * Linked list node.
 */
struct list {
	/** Next node. */
	struct list *next;

	/** Data. */
	list_data_t data;
};
