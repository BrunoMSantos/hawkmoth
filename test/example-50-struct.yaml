directive-arguments:
- example-50-struct.c
