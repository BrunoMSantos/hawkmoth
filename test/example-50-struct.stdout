.. c:type:: struct list

   Linked list node.
   

.. c:member:: struct list * list.next

   Next node. 
   

.. c:member:: int list.data

   Data. 
   

